packaging/installer/kickstart.sh

```diff
@@ -2134,7 +2134,7 @@ parse_args() {
       "--claim-only") set_action 'claim' ;;
       "--no-updates") NETDATA_AUTO_UPDATES=0 ;;
       "--auto-update") NETDATA_AUTO_UPDATES="1" ;;
-      "--auto-update-method")
+      "--auto-update-type")
```

Comment:
```suggestion
      "--auto-update-type|--auto-update-method")
```

Needs to keep parsing the old name as well for now, this is a potentially breaking change and continuing to support the old option name costs us essentially nothing. I’ll set a reminder to add a warning to the old option name in a few weeks and remove it completely in a couple of months.

Reply:
I don't see why we need that, `--auto-update-method` was never exposes to users:
- not in docs 
- not in the script usage.

Not a breaking change.

Reply:
Ok, I see we have been having it wrong since v1.34.0.

```bash
netdata ((v1.34.0) *%) $ git grep "auto-update-method"
packaging/installer/kickstart.sh:    "--auto-update-method"
```

Reply:
updated